input/fsh/OrganizationSECommonsLite.fsh

```diff
@@ -84,4 +84,12 @@ Description: """This is a light weight Swedish Organization profile.
 
 * address.line.extension contains
             iso21090-ADXP-careOf named careOf 0..* MS
-        and iso21090-ADXP-streetAddressLine named streetAddressLine 0..* MS
\ No newline at end of file
+        and iso21090-ADXP-streetAddressLine named streetAddressLine 0..* MS
+
+Instance: OrganizationSELiteExample
```

Comment:
Which style we'll follow? I preper 2nd option :-)
1. Have all the examples in the profile definition fsh file?
2. maintain examples separately in separate files in ./input/fsh/example folder?

Reply:
I have a preference for the first, but it's not a very strong one.

Reply:
If the second one is being used more wide I okay with that,